packages/grafana-toolkit/package.json

```diff
@@ -28,10 +28,10 @@
   "dependencies": {
     "@babel/core": "7.9.0",
     "@babel/preset-env": "7.9.0",
-    "@grafana/data": "next",
+    "@grafana/data": "7.5.0-pre.0",
```

Comment:
I wonder if we had specified this as ^7.0.0, wouldn't it by default resolve to workspace package?

Reply:
Would be good to see why https://github.com/grafana/grafana/pull/30428 did not update toolkit's dependencies

Reply:
> I wonder if we had specified this as ^7.0.0, wouldn't it by default resolve to workspace package? Asking because otherwise, we will have to update the version string with each release.

I believe Lerna (which uses yarn workspaces under the hood) will only bump versions if the dependency matches the workspace version. Otherwise it will treat it as a package to pull from the registry. As noted [here](https://classic.yarnpkg.com/en/docs/workspaces/#toc-limitations-caveats) in the yarn docs...

>In the example above, if workspace-b depends on a different version than the one referenced in workspace-a’s package.json, the dependency will be installed from npm rather than linked from your local filesystem. This is because some packages actually need to use the previous versions in order to build the new ones (Babel is one of them).

> Would be good to see why #30428 did not update toolkit's dependencies

I kind of confirmed this by running the `yarn packages:prepare` script in master and compared it to running the same script in this branch. This branch will bump the versions used by grafana-toolkit whilst in master it leaves them untouched.

Reply:
I also understood that they had to match exactly.